frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/material-popup/material-popup.component.html

```diff
@@ -0,0 +1,18 @@
+<ng-container *ngIf="material$ | async as material">
+    <h5 class="popover-header">
+        {{ material.vehicleName }}: Material
+        <button
+            (click)="closePopup.emit()"
+            type="button"
+            class="btn-close float-end"
+        ></button>
+    </h5>
+    <div class="popover-body">
+        <div class="mb-3">
```

Comment:
I think you can remove this div and the `mb-3`.
The space on the bottom should just be the padding from the `popover-body`.
If something else is later added under the `app-cater-capacity` you can add the padding back in.

Btw. you can also style angular component selectors like 
```ts
<app-my-component class="mb-3"></app-my-component>
``` 
the only exceptions are (I believe) all `ng-` components (`ng-container`, `ng-content`, `ng-template`).

Reply:
Same for the other popup.

Reply:
Looks a lot better, thank you ^^